tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -223,4 +225,5 @@ def extension_modules():
         'grpcio>={version}'.format(version=grpc_version.VERSION),
     ],
     package_data=package_data(),
+    test_suite=_TEST_CLASS,
```

Comment:
If we already testing with Bazel, do we still need this?

Reply:
Not necessarily, but since we're already testing under both frameworks in the rest of the project, I figured it was best to follow the same pattern here. I'll remove if you feel strongly about it.

Reply:
Are we testing it through `run_tests.py`? If not, I would recommend to remove it. It could reduce our burden by implicitly saying those tests are meant for Bazel.